gnomad_constraint/resources/resource_utils.py

```diff
@@ -29,6 +29,7 @@
 )
 
 
+
```

Comment:
was this extra line intentional? it looks like there are already two blank lines above